src/middlewares/auth/isAuthorized.js

```diff
@@ -60,19 +60,22 @@ const isAuthorized =
 
     switch (true) {
       case page == null && action == null:
+        //it allows users who has valid access token
         next();
         break;
 
       case process.env.IS_AUTHORIZED_CHECK === "DISABLE":
+        //it disables the authoraisation check
         next();
         break;
-
       default:
         try {
+          //it allows bussiness admin with required permission to the page
           await checkIsAuthorized(req, res, next, { page, action, options });
           next();
         } catch (err) {
           try {
+            //it allowshandle for parent
```

Comment:
update the comment message in code `it allows/restrict the access of resource for a parent`